app/assess/routes.py

```diff
@@ -28,89 +26,76 @@
     "/application_id/<application_id>/sub_criteria_id/<sub_criteria_id>",
     methods=["POST", "GET"],
 )
-@login_required(roles_required=["COMMENTER"])
 def display_sub_criteria(
     application_id,
     sub_criteria_id,
 ):
     """
     Page showing sub criteria and themes for an application
     """
-    role_information = {
-        "is_commenter": g.user.highest_role == "COMMENTER",
-        "scoring_permissions": any(
-            required_role in g.user.roles
-            for required_role in ["ASSESSOR", "LEAD_ASSESSOR"]
-        ),
-    }
-    
-    displayCommentBox = False
-    if 'add-comment' in request.args.keys() and request.args['add-comment'] == '1':
-        displayCommentBox = True            
```

Comment:
Is this being unnecessarily removed?

Reply:
Maybe a botched merge conflict resolution?

Reply:
role_information dict has been replaced as flask g is now available globally as a template variable

Reply:
display_comment_box functionality has been simplified and moved to logic on line 113-134